### DISCUSSION TITLE
6627a display

### DISCUSSION THREAD

**Discussion by superDIYer** [xyphro/HP3457-OLED-display#7] [General]
Got a new hp power supply that has a very familiar looking display on it. I think its a derivative of the one for the 3457a, an impression confrimed by the note on the Readme for this project. May try replacing them both!

**Comment by xyphro** [comment]
Let us know if it works, I am curious! If you see in service manuals that the oart number is the same it is very likely to work. Otherwise maybe some minor sw changes will be required. We'll see!

**Comment by superDIYer** [comment]
Here are pics of the back of the 6627a display board. 

Board part number
![20240223_225709](https://github.com/xyphro/HP3457-OLED-display/assets/78966282/d906c902-b45b-466a-b439-e76a0f142395)

Ribbon cable connection 
![20240223_225716](https://github.com/xyphro/HP3457-OLED-display/assets/78966282/c6fdd076-ee8a-4ab2-a67f-aa60a199c963)

**Comment by superDIYer** [comment]
According to the "xi" original post, the replacement display is being compared to the 5181-2813... so I think we're really and truly gonna luck out here.
